File: CodeForces/701-800/791/A/791A.py
# Problem 791A: Bear and Big Brother
# https://codeforces.com/contest/791/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: October 23rd, 2025
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

limak, bob = map(int, input().split())
year = 1
while True:
    limak *= 3
    bob *= 2
    if limak > bob:
        print(year)
        break
    else:
        year += 1

File: CodeForces/701-800/791/A/791A.php
<?php

// Problem 659C: Tanya and Toys
// https://codeforces.com/contest/659/problem/C
// Executed with: PHP 8.4.11 (cli) (built: Aug 13 2025 01:43:48) (NTS)
// Linux version: Linux kernel 6.17.0-5-generic
// Written on: October 27th, 2025
// Codeforces language used: Cannot be submitted

$arr = array_map('intval', explode(' ', trim(fgets(STDIN))));
$limak = $arr[0];
$bob = $arr[1];

$year = 0;
while (true) {
    if ($limak > $bob) {
        break;
    }
    $limak *= 3;
    $bob *= 2;
    $year += 1;
}

echo $year . "\n";

?>
